### PR TITLE
Update README.md #1972

### PR DESCRIPTION
**Pull request by seekaddo**
Updated the README.md for the current JtR.

### CODE REVIEW

**Comment by seekaddo** [comment]
@magnumripper how do i remove the pull request from the previous problem

> c4dc856
>  @seekaddo
> fixed linesfeed
> 3dd5b68

I tried git reset before creating this pull request but it comes with 3 files instead of one. OMG

**Comment by kholia** [comment]
Quick comments,

1. Change `jtr` to `JtR` or `john`.

2. Same goes for occurrences of `Jtr`. Yes, consistency is important :)

3. For `README.gpg - PGP Desktop / OpenPGP / GnuPG private cracking in jtr` entry, replace `PGP Desktop` with `PGP Zip`.

4. For `IOS 7`, correct the casing.

5. `dumping apex hashes` -> `dumping Oracle APEX...`

I will try finding more later.

**Comment by seekaddo** [comment]
@kholia Thanks for the feedback. Right on it.

**Comment by seekaddo** [comment]
@kholia Thanks. this git giving me a cloudy day. 😀

**Comment by kholia** [comment]
More fixes,

* s/prince/PRINCE

* s/opencl/OpenCL

* s/encrypted volumes in jtr/encrypted volumes with JtR

* s/ssh/SSH (not entirely sure about this)

**Comment by kholia** [comment]
Change commit message -> `Updated README for the current jtr` -> `Update README.md for the current JtR`.

**Comment by seekaddo** [comment]
@kholia  do you mean src/ with this

> More fixes,
> 
> s/prince/PRINCE

**Comment by seekaddo** [comment]
@kholia It worked OMG. Thanks, man.

**Comment by kholia** [comment]
> Should I change the description for this docs?

Just change the description text in the `README.md` file.

Also, I would just modify the `README.md` file in this PR. We can fix the various documents in a separate PR, if required.

**Comment by seekaddo** [comment]
> Also, I would just modify the README.md file in this PR. We can fix the various documents in a separate PR, if required.

Yea of course, I had not intention modifying the original files. Sorry for the misunderstanding.

File: README.md [hunk 1]
```diff
@@ -170,6 +170,70 @@ the recommended order of reading:
...
+* RULES-hashcat - wordlist rules with hashcat extension
+* Regen-Lost-Salts.txt - regen-lost-salt in JtR
+* SIPcrack-LICENSE - the SIPcrack license
+* SecureMode-tutorial.md - using JtR's SecureMode feature
+* dynamic_history.txt - upto date history on dynamic_fmt.c file
+* john-1.7.9-jumbo-7-licensing-stats.txt - license status for john-1.7.9-jumbo-7
+* libFuzzer-HOWTO.txt - how to build libfuzzer
+* pass_gen.Manifest - pass_gen.pl version history
+* pcap2john.readme - all the prior copyright headers from the independent XXX2john.py PCAP conversion utilities
+* AxCrypt-Auditing-HOWTO - auditing AxCrypt secrets
```
Comment (magnumripper) [inline README.md]: This document says "recommended order of reading" at the top. Then this commit starts with "AddressSanitizer-HOWTO.txt" which is really a super-niched thing for developers only.

There are so many documents we might need to drop the "recommended order" clause, or break it with a "rest of documents in alphabetical (or whatever) order"

Comment (seekaddo) [reply README.md]: @magnumripper hahahaha. Right. I will rearrange it. But i think with the readme cracking stuff, I can create another little for that and group them in alphabetical order.

**Comment by seekaddo** [comment]
@magnumripper I have updated the suggested changes

**Comment by magnumripper** [comment]
I can live with this. I can also think of improvements if we wish: We could add a "For developers" heading and group stuff like that there. And perhaps more headings, although I can't think of one right now.

**Comment by magnumripper** [comment]
I'm merging this for now, thanks!

**Comment by seekaddo** [comment]
@magnumripper Why can we create something small for developers or new contributors. Like a small wiki and a little git command helpful links. I am really ashamed of all the little mistakes I make in here but I still learn a lot more from you guys. I think we need this for JtR. I can help on that if I get little guidelines on there.

I also think we should separate the contributor or developer link information from the README page. There should be a full page for that. @solardiz @jfoug What do you guys think?

File: README.md [hunk 2]
```diff
@@ -170,9 +170,77 @@ the recommended order of reading:
...
+* README.MinGW - Fedora >= 22 cross-compiling instructions
+* README.mozilla - cracking Mozilla Firefox, Thunderbird and SeaMonkey master passwords
+* README.mpi - using MPI in JtR
+* README-OPENCL - how to use opencl in JtR
+* README-PDF - PDF cracking in JtR
+* README-PST - PST cracking in JtR
+* README.pwsafe - cracking Password Safe 3.x and Password Gorilla databases with john
+* README.ssh - Cracking password protected ssh private keys
+* README-ZIP - ZIP cracking in JtR
+* README-ZTEX - using ZTEX with JtR
```
Comment (solardiz) [inline README.md]: ZTEX is a company. One can't really use the company with JtR. We could say "using ZTEX FPGA boards with JtR".

Comment (seekaddo) [reply README.md]: lol, didn't know that. I will correct this error.

File: README.md [hunk 3]
```diff
@@ -170,9 +170,77 @@ the recommended order of reading:
...
+* Regen-Lost-Salts - regen-lost-salt in JtR
+* RULES-hashcat - wordlist rules with hashcat extension
+* SecureMode-tutorial - using JtR's SecureMode feature
+* SIPcrack-LICENSE - the SIPcrack license
+
+
 (*) most users can safely skip these.
```
[the remark this answers is not in the document]
Comment (seekaddo) [reply README.md]: I will do that.

**Comment by seekaddo** [comment]
@solardiz sorry, I have made those changes now. Can @magnumripper revert this so that i add this little changes to it before we merge again?

**Comment by magnumripper** [comment]
> @magnumripper Why can we create something small for developers or new contributors. Like a small wiki and a little git command helpful links. I am really ashamed of all the little mistakes I make in here but I still learn a lot more from you guys. I think we need this for JtR. I can help on that if I get little guidelines on there.

That's a good idea. I'm not really comfortable with the Openwall wiki, it has an interface that doesn't really work for me. Still, it's probably what we should use. Feel free to have a good look at the Openwall wiki and start cleaning things up there!

**Comment by solardiz** [comment]
> I also think we should separate the contributor or developer link information from the README page. There should be a full page for that. @solardiz @jfoug What do you guys think?

You're probably right. I currently have no preference on this.

**Comment by magnumripper** [comment]
> @solardiz sorry, I have made those changes now. Can @magnumripper revert this so that i add this little changes to it before we merge again?

It's not that problematic. You can open a new PR with further enhancements!

**Comment by seekaddo** [comment]
> It's not that problematic. You can open a new PR with further enhancements!
@magnumripper ok will do that. 


> Feel free to have a good look at the Openwall wiki and start cleaning things up there!
Will take look. Thanks.

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (solardiz) [reply README.md]: Specifically, the "(*) most users can safely skip these." comment should be moved to be above the jumbo-specific list of files, and any asterisks on those jumbo-specific files removed.

**Comment by solardiz** [comment]
As far as I can tell, my feedback on this PR has been ignored so far. @seekaddo was going to open a new PR to address it, but never did. :-(

**Pull request merged**